text/processing.py

```diff
@@ -202,3 +204,28 @@ def run(self):
             self.lines = int(length * self.ratio)
 
         self.sh("zcat -f {text_file} | tail -n {num_lines} | gzip > {out}")
+
+
+class DiffJob(Job):
```

Comment:
I think we better name it differently (e.g. `SetDifferenceJob`) because after `HeadJob` and `TailJob` on might assume that `DiffJob` would be a wrapper around the GNU diff utillity.